### PR TITLE
Task#38159 Finish Task 6

### PR DESCRIPTION
**Pull request by SUN-nguyen-van-tai-b**
link: https://edu-redmine.sun-asterisk.vn/issues/38159

### CODE REVIEW

**Comment by SUN-nguyen-van-tai-b** [comment]
ready

File: src/pug/pages/detail.pug [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+              span 1 Review(S) | Add Your Review
+          .info__color
+            h3.info__color-label màu sắc
+            ul.info__color-list
+              li.list__item
+              li.list__item.list__item--dark
+              li.list__item.list__item--red
+          .info__sizes
+            h3.info__sizes-label kích cỡ
+            select
```
Comment (manhnt-0960) [inline src/pug/pages/detail.pug]: cho vào form chưa e

File: src/pug/pages/detail.pug [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            img(src="./assets/images/home-image7.jpg", alt="products-7" class="image__wrapper-main")
+            img(src="./assets/images/ruounho.png", alt="ruounho-7" class="image__wrapper-inner")
+  section.relate
+    .container
+      +heading("Sản phẩm liên quan")(class="heading--relate")
+      .newproduct__list
+        each item in [1,2,3,4]
+          +renderProduct(`./assets/images/products/${item}.jpg`,"rượu vang đà lạt", "370.000","450.000",)
+
+
```
Comment (manhnt-0960) [inline src/pug/pages/detail.pug]: xóa dòng thừa e

File: src/pug/pages/detail.pug [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+        .detail__review-image
+          .image__wrapper
+            img(src="./assets/images/home-image7.jpg", alt="products-7" class="image__wrapper-main")
+            img(src="./assets/images/ruounho.png", alt="ruounho-7" class="image__wrapper-inner")
+  section.relate
+    .container
+      +heading("Sản phẩm liên quan")(class="heading--relate")
+      .newproduct__list
+        each item in [1,2,3,4]
+          +renderProduct(`./assets/images/products/${item}.jpg`,"rượu vang đà lạt", "370.000","450.000",)
```
Comment (manhnt-0960) [inline src/pug/pages/detail.pug]:
```suggestion
          +renderProduct(`./assets/images/products/${item}.jpg`, "rượu vang đà lạt", "370.000", "450.000")
```

**Comment by SUN-nguyen-van-tai-b** [comment]
ready

**Pull request merged**